apps/bb2_tools/migrations/0002_v2user.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.24 on 2021-07-02 17:59
```

Comment:
NOTE: This migration file is related to the merged in BB2-146 migrations squash PR. So including it here.